templates/terraform/dns/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable dns_names {
+  type = list(string)
+  description = "List of DNS names to generate global IP addresses, A-records, and CNAME-records for."
+}
+
+variable dependencies {
+  type = any
```

Comment:
Should this be `type = list(any)`? If so, then you could probably remove the brackets that convert this variable to a list elsewhere.

Reply:
Talked about this offline.